build.gradle

```diff
@@ -142,223 +159,209 @@ task cucumberTestReport(type: TestReport) {
-    compile "com.icthh.xm.commons:xm-commons-metric:${xm_commons_version}"
-    compile "com.icthh.xm.commons:xm-commons-logging-web:${xm_commons_version}"
-}
-
-checkstyle {
-    toolVersion = "7.6.1"
-    ignoreFailures = true
-    configFile = rootProject.file('./qa/checkstyle.xml')
-}
+    // XM custom dependencies
+    implementation "org.hibernate:hibernate-envers"
+    implementation 'com.vladmihalcea:hibernate-types-52:2.4.3'
+    implementation 'com.vladmihalcea:hibernate-types-52:2.4.3'
 
-findbugs {
-    toolVersion = '3.0.1'
-    ignoreFailures = true
-}
+    compileOnly "org.projectlombok:lombok:${lombok_version}"
+    annotationProcessor "org.projectlombok:lombok:${lombok_version}"
+    annotationProcessor("javax.xml.bind:jaxb-api")
+
+    implementation "io.dropwizard.metrics:metrics-json"
+    implementation "io.dropwizard.metrics:metrics-jvm"
+    implementation "io.dropwizard.metrics:metrics-servlet"
+    implementation "io.dropwizard.metrics:metrics-servlets"
+
+    implementation "com.icthh.xm.commons:xm-commons-timeline:${xm_commons_version}"
+    implementation "com.icthh.xm.commons:xm-commons-security:${xm_commons_version}"
+    implementation "com.icthh.xm.commons:xm-commons-migration-db:${xm_commons_version}"
+    implementation "com.icthh.xm.commons:xm-commons-tenant-endpoint:${xm_commons_version}"
+    implementation "com.icthh.xm.commons:xm-commons-permission:${xm_commons_version}"
+    implementation "com.icthh.xm.commons:xm-commons-ms-web:${xm_commons_version}"
+    implementation "com.icthh.xm.commons:xm-commons-lep:${xm_commons_version}"
+    implementation "com.icthh.xm.commons:xm-commons-i18n:${xm_commons_version}"
+    implementation "com.icthh.xm.commons:xm-commons-metric:${xm_commons_version}"
+    implementation "com.icthh.xm.commons:xm-commons-logging-web:${xm_commons_version}"
+    implementation "com.icthh.xm.commons:xm-commons-metric:${xm_commons_version}"
 
-pmd {
-    rulePriority = 3
-    toolVersion = '5.5.4'
-    ignoreFailures = true
-    ruleSetFiles = files("./qa/pmd.xml")
-}
+    testCompileOnly "org.projectlombok:lombok:${lombok_version}"
+    testAnnotationProcessor "org.projectlombok:lombok:${lombok_version}"
+    liquibaseRuntime "com.h2database:h2"
+    testImplementation(group: 'org.testcontainers', name: 'testcontainers', version: '1.10.5') {
+        exclude group: 'commons-io', module: 'commons-io'
+    }
+    testImplementation group: 'org.testcontainers', name: 'postgresql', version: '1.10.5'
 
-clean {
-    delete "target"
+    testImplementation("com.github.database-rider:rider-spring:1.2.7") {
+        exclude module: 'slf4j-api'
+        exclude module: 'slf4j-simple'
+    }
 }
 
 task cleanResources(type: Delete) {
-    delete 'build/resources'
+    delete "build/resources"
 }
 
 wrapper {
-    gradleVersion = '4.10.2'
+    gradleVersion = "6.0.1"
 }
 
 task stage(dependsOn: 'bootWar') {
 }
 
 bootWar.dependsOn war
-compileJava {
```

Comment:
better to leave this section in the same place

Reply:
Ok